subcommands/keys/tuf_updates_rotate_offline_key.go

```diff
@@ -213,9 +217,14 @@ func replaceOfflineRootKey(
 func replaceOfflineTargetsKey(
 	root *client.AtsTufRoot, onlineTargetsId string, creds OfflineCreds, keyType TufKeyType,
 ) (*TufSigner, OfflineCreds) {
+	oldKey, err := FindOneTufSigner(root, creds,
+		subcommands.SliceRemove(root.Signed.Roles["targets"].KeyIDs, onlineTargetsId))
```

Comment:
Why are we replacing the online key? We may wanna rotate the exiting offline targets key too?

Reply:
I exclude an online key from a list of keys to search for.
This warrants that the online key is never replaced, even if a user manually inserted it into the keys file.